code/modules/mob/new_player/new_player.dm

```diff
@@ -186,8 +186,9 @@
 			src << browse(null, "window=playersetup")
 			spawning = TRUE
 			stop_sound_channel(CHANNEL_LOBBYMUSIC)
-
-
+			if(SSticker.current_state < GAME_STATE_PLAYING)
```

Comment:
I'd say you should give this a 1-5 minute timer for ghosts to join imo. Maybe 2 minutes

Reply:
Ehh, maybe in a later PR. For now, just roundstart is what's been approved.